src/devenv.nix

```diff
@@ -78,9 +78,9 @@ pkgs.writeScriptBin "devenv" ''
         echo "No 'processes' option defined."  
         exit 1
       else
-        echo "Starting processes ..." 1>&2
-        echo "" 1>&2
-        ${pkgs.honcho}/bin/honcho start -f $procfile --env $procfileenv
+        procfilescript=$($CUSTOM_NIX/bin/nix $NIX_FLAGS build --no-link --print-out-paths --impure '.#procfileScript')
```

Comment:
This should be done at line 73 and we don't need procfile & procfileenv anymore

Reply:
Is my latest push what you were thinking? I ended up having to remove the check for if processes were defined since it uses `$procfile`